### DISCUSSION TITLE
测试发现OpenClash可以加速WIFI连接带宽测速的速率，为什么会这样呢？

### DISCUSSION THREAD

**Discussion by warcns** [vernesong/OpenClash#2815] [Q&A]
先说本地网络配置，小米AX3600是主路由器，OpenClash安装在旁路由上，OpenClash使用Redir模式，打开**实验性：绕过中国大陆 IP**功能。

发现PC网线连接主路由器，网关走的旁路由OpenClash，打开**实验性：绕过中国大陆 IP**功能，测速本地带宽可以达到700-800mbps，但是MAC 使用WIFI连接，网关走旁路由的话本地测速一般在500-600mbps，测试MAC WIFI直连主路由器，不走OpenClash，速度也是在500-600mbps。

但是当WIFI连接主路由，网关走旁路由OpenClash，并关闭**实验性：绕过中国大陆 IP**功能后，发现网速测试直接达到700-800mbps左右，然后同样的方法测试PS5和iPhone 14 PM的网速，也是一样的情况。关闭**实验性：绕过中国大陆 IP**，网关走OpenClash后可以把WIFI连接的网速带宽从500左右提升到700左右。

想请问一下各位大佬能否分析一下这是什么原因呢，我是希望能打开**实验性：绕过中国大陆 IP**这项功能，提升国内的UDP转发的，但关闭这个功能后又会明显提升WIFI的带宽速度，没想明白是为什么。

测速截图，上面两个700+的关闭**实验性：绕过中国大陆 IP**，走OpenClash网关的Wi-Fi测速。下面两个400+的一个是打开**实验性：绕过中国大陆 IP**走OpenClash的，另外一个是直接走主路由器AX3600，没过旁路由的测速。

理论上WIFI连接走主路由测试应该是最快的，但偏偏是WiFi连接主路由，然后网关走OpenClash旁路由后反倒更快，实在没想明白是为什么。

<img width="366" alt="截屏2022-11-15 01 35 23" src="https://user-images.githubusercontent.com/56870885/201728318-7e1d41fa-8566-40c9-8988-a33575bad3d4.png">

**Comment by omg7789** [comment]
跟你一样是ax3600
用openwrt当主路由使用
不存在这个问题
猜测开启绕过大陆ip后 主路由和旁路由来回折腾造成的损耗
你可以试一下网关指向主路由
旁路由打开绕过 插线向交换机和不插线的对比

**Reply by warcns** [reply]
我WIFI直连主路由器的速度都没有走OpenClash的速度快。。。所以才疑惑的。
